projects/storefrontlib/src/cms-components/checkout/components/shipping-address/shipping-address.component.ts

```diff
@@ -27,6 +27,7 @@ export class ShippingAddressComponent implements OnInit {
   existingAddresses$: Observable<Address[]>;
   newAddressFormManuallyOpened = false;
   isLoading$: Observable<boolean>;
+  changeSelectedAddressLoaderState$: Observable<any>;
```

Comment:
2 comments:
- For variable naming: Since we are setting the default address, this is probably more a process state than a loader state.
- Observable<any> : Let's replace any with the appropriate type.